ntcore/src/test/native/cpp/SpanMatcher.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#ifndef NTCORE_SPANMATCHER_H_
+#define NTCORE_SPANMATCHER_H_
```

Comment:
Why this style of header guard and not `#pragma once` which is used everywhere else?

Reply:
This part of the library mostly had this style.  I will update for consistency.